tests/run_tests.py

```diff
@@ -1,37 +1,45 @@
 #!/usr/bin/env python
-'''VIC command line testing interface'''
+'''VIC testing command line interface'''
 
 from __future__ import print_function
 import os
+import sys
 import glob
 import argparse
 import datetime
 from collections import OrderedDict
 import string
-from tonic.models.vic.vic import VIC, VICRuntimeError, read_vic_ascii
+
+import pytest
+import pandas as pd
+
+from tonic.models.vic.vic import VIC, VICRuntimeError  # , read_vic_ascii
```

Comment:
We need to update tonic's `read_vic_ascii` for VIC 5 so for now I'm just using the function at the bottom of this script.